src/composables/accounts.ts

```diff
@@ -53,6 +71,13 @@ const mnemonic = useStorageRef<string>(
       ...((IS_IOS && IS_MOBILE_APP) ? [migrateMnemonicCordovaToIonic] : []),
       migrateMnemonicVuexToComposable,
     ],
+    onRestored: async (val) => {
+      const hasStoredMnemonic = (
+        await WalletStorage.get(STORAGE_KEYS.mnemonic)
+        || await SecureMobileStorage.get(STORAGE_KEYS.mnemonic)
+      );
```

Comment:
The storageRef composable is doing this when restoring the value. Why we need to do it again here?